src/generate/config.mdx

```diff
@@ -6,237 +6,199 @@ route: /projectmanagement/config
-
-        // Should the overlay be enabled
-        "enabled": true,
-
-        // Font to be used when showing text on the overlay .
-        // Please see the FAQ for supported file types.
-        "font_path": "./fonts/Lato-Regular.ttf",
-
-        // Name / Title of the stream
-        "title": {
-          
-          // See Common Text Object Settings
-          
-          // Title Unique Settingss
-
-          // Should the title scroll across the screen.
-          "enable_scroll": true,
-
-          // Speed at which the title scrolls
-          "font_scroll_speed": "20",
-        },
-
-        // Artist of the Current Playing Song, from the file metadata
-        "artist": {
-          
-          // See Common Text Object Settings
-
-        },
-
-        // Album of the Current Playing Song, from the file metadata
-        "album": {
-
-          // See Common Text Object Settings
-
-        },
-
-        // Title of the Current Playing Song, from the file metadata
-        "song": {
-
-          // See Common Text Object Settings
-
-        },
-
-        // Image overlay shown on top of the text and the video
-        "image": {
-
-          // See Common Image Object Settings
-
-        }
-      }
-    },
-
-    // Settings reffering to interludes that are occasionally played along with radio songs
-    "interlude": {
-
-      // Should interludes be played
-      "enabled": "true",
-
-      // How often should interludes by played. This is a non-negative float representing a percentage.
-      // Valid values are from 0.0 (never) - 1.0 (100% of the time)
-      "frequency": "0.2",
-
-      // Directory where Interlude audio is stored.
-      // Can be the same as the radio directories.
-      "audio_directory": "./interludes/audio",
-
-      // Directory where Interlude video is stored. 
-      // Can be the same as the radio directories.
-      "video_directory": "./interludes/video",
-
-      // Overlay that is shown above interlude streams
-      "overlay": {
-
-        // Should the overlay be shown
-        "enabled": true,
-
-
-        // Please see the FAQ for supported file types.
-        "font_path": "./fonts/Lato-Regular.ttf",
-
-        // See Common Text Object Settings
-
-        "title": {
-
-           // See Common Text Object Settings
-
-        },
-        "image": {
-
-          // See Common Image Object Settings
-
-        }
-      }
-    }
-}
-```
-
-## Common Text Object Settings
-
-Common keys that all text objects support and are required to display the text
-
+This page will go in detail with every field of a projects `config.json`. If you ever need to reset the default settings, please check out the [default config.json](https://github.com/torch2424/live-stream-radio/blob/master/src/generate/template/config.json)
+
+## Settings Fields
+
+| Field | Usage |
+| - | - |
+| stream_url | The URL of the stream. You can use $stream_key as a variable and overwrite it with the `--output` CLI flag. |
```

Comment:
Can you make it a little more clear that the $stream_key is replaces by the stream_key field?

In retrospect, should have just made it one field, but most Stream providers separate the logic, so just trying to follow that. Maybe in a later update we can do that 👍

Reply:
This is the only real change I want, the rest are compliments haha!